dist/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/54def629d4.js" crossorigin="anonymous"></script>
+    <title>Foodie App</title>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+  <header>
+    <nav id="page-nav">
+      <a href="#"><img src="." alt="Restaurant-Logo" class="page-img"></a>
+
+      <ul class="nav-links">
```

Comment:
- [ ] Let's display a count of the returned items from the API on the nav, as shown on the wireframe